graphh/CGH.py

```diff
@@ -48,42 +51,84 @@ class GraphHopper(object):
                                   'time': 67674},
     ...
     """
+
+
     url = "https://graphhopper.com/api/1/"
 
     def __init__(self, api_key, premium=False):
+
         self.api_key = api_key
         self.premium = premium
 
-    def _url_handle(self, api, l_parameters):
-        """This function does an url request with given parameters
+
+
+    def _url_requests(self, api, data, request="get"):
+
+        """ This function does an url request ( GET or POST) with given parameters
 
         Parameters
         ----------
-         api: str
+        api: str
             name of the api used
-         l_parameters: list
-            list of parameters to insert in the url
+        data: dict
+            dict of parameters to insert in the url
 
```

Comment:
Please add the `request` argument to the docstring and document it properly.

Reply:
There is no `s` at the end of `request`: be careful when you name arguments: what you write here is used to generate the docs.